src/proto/streams/streams.rs

```diff
@@ -442,6 +483,10 @@ where
         let me = &mut *me;
 
         let stream = me.store.resolve(self.key);
+        if stream.is_pending_open {
+            //XXX: no?
```

Comment:
This differs from the behavior when the user has an opened `Stream`, but tries to send more data than there is capacity in the window. We could stick with this (with a different error variant), or allow buffering data even on a pending stream, to be more consistent for the user.